app/database_etl/summary_report_generator/summary_report.py

```diff
@@ -0,0 +1,64 @@
+from time import time
+from app.serotracker_sqlalchemy import db_session, DashboardSource, ResearchSource, \
+    Country, State, City, TestManufacturer, AntibodyTarget, CityBridge, StateBridge, \
+    TestManufacturerBridge, AntibodyTargetBridge
+from sqlalchemy import func, inspect
+from app.utils.notifications_sender import send_slack_message
+import json
+import datetime
+
+class SummaryReport:
+    def __init__(self):
+        self.start_time = time()
+        self.get_table_counts_before()
+
+    def get_elapsed_time(self):
+        return time() - self.start_time
+
+    def get_num_airtable_records(self, num_records: int):
```

Comment:
shouldn't this be set_num_airtable_records?

Reply:
yes agree